### PR TITLE
BUG: Fix Incorrect handling of project rules.

### PR DESCRIPTION
**Pull request by naterichman**

### CODE REVIEW

File: tests/requirements.txt [hunk 1]
```diff
@@ -2,5 +2,6 @@ pydicom~=2.0.0
 pytest
 pytest-cov
 pytest-mock
-flywheel-sdk~=11.0.0
+flywheel-sdk>=11.0.0
+flywheel_gear_toolkit==0.1.1
```
Comment (NPann) [inline tests/requirements.txt:6]: Needed ?

**Review by NPann — APPROVED** [review]
LGTM. But that additional gear-toolkit dep.

File: tests/requirements.txt (hunk 1 above)
Comment (naterichman) [reply tests/requirements.txt:6]: Yes

Comment (naterichman) [reply tests/requirements.txt:6]: actually moved gear toolkit from install deps to test deps

**Pull request merged**